egs/aishell/s10/chain/model.py

```diff
@@ -93,20 +92,17 @@ def __init__(self,
         tdnnfs = []
         for i in range(num_layers):
             time_stride = time_stride_list[i]
-            conv_stride = conv_stride_list[i]
             layer = FactorizedTDNN(dim=hidden_dim,
                                    bottleneck_dim=bottleneck_dim,
-                                   time_stride=time_stride,
-                                   conv_stride=conv_stride)
+                                   time_stride=time_stride)
             tdnnfs.append(layer)
 
         # tdnnfs requires [N, C, T]
         self.tdnnfs = nn.ModuleList(tdnnfs)
 
         # prefinal_l affine requires [N, C, T]
         self.prefinal_l = OrthonormalLinear(dim=hidden_dim,
-                                            bottleneck_dim=bottleneck_dim * 2,
-                                            time_stride=0)
+                                            bottleneck_dim=bottleneck_dim * 2)
```

Comment:
mm.. I'm a bit surprised this `* 2` is here?

Reply:
mm.. I think you were assuming that the final layer's bottleneck is always twice the TDNN-F layers' bottleneck.
In fact we generally leave the final layer's bottleneck at 256, which for some reason seems to work across a range
of conditions.  You could make that a separate configuration value.

Reply:
when I have checked the param shape of kaldi's model, I don't find the difference betwieen the final layer and previous layers what you said.
```
'tdnnf12.linear':'time_offsets': array([-3,  0]), 'params': (128, 2048)
'tdnnf12.affine':'time_offsets': array([0, 3]), 'params': (1024, 256)
'tdnnf13.linear':'time_offsets': array([-3,  0]), 'params': (128, 2048)
'tdnnf13.affine':'time_offsets': array([0, 3]), 'params': (1024, 256)
'prefinal-l':'params': (256, 1024)
```

Reply:
> mm.. I'm a bit surprised this `* 2` is here?

`* 2` is used here to follow what kaldi does.
I've changed it to be configurable in this pullrequest: https://github.com/kaldi-asr/kaldi/pull/3925